article/main.tex

```diff
@@ -42,8 +42,8 @@
 \usepackage{blindtext}
 
 % Declare article info
-\title{Some Title}
-\author{All of us}
+\title{Creating, testing, and using causal graphs in choice modelling}}
+\author{Timothy Brathwaite, Mohamed Amine Bouzaghrane, Hassan Obeid}
```

Comment:
Suggest the ordering as "Timothy Brathwaite, Hassan Obeid, Mohamed Amine Bouzaghrane" to reflect contributions and the process of coming up with the vision/plan for the work.

Reply:
Let's discuss this further! I'll leave the authors blank for now and merge the PR with just the changed title.